style.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color: blue;
+}
```

Comment:
[Required ] as mentioned in the requirement style.css must include one class with styles for your header but instead, you just select all h1 elements please use classes instead 
because CSS classes enable you to apply unique style properties 
check this [article](https://blog.hubspot.com/website/what-is-css-class)